<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Docking</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>15</y>
     <width>386</width>
     <height>444</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <layout class="QHBoxLayout" name="radioLayout">
      <item>
       <widget class="QRadioButton" name="pdbFileRadio">
        <property name="text">
         <string>Choose from file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pymolObjectRadio">
        <property name="text">
         <string>Choose from PyMOL object</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="pymolObjectLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>PyMOL object</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <layout class="QHBoxLayout" name="pymolObjectLayout">
      <item>
       <widget class="QComboBox" name="ligand_combobox">
        <item>
         <property name="text">
          <string>ligand</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="receptor_combobox">
        <item>
         <property name="text">
          <string>receptor</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshObjectsButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>‚ü≤</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="pdbFileLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Ligand PDB file</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="FileSelector" name="pdbFileSelector">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="pdbFileLabel_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Receptor PDB file</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="FileSelector" name="pdbFileSelector_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QLabel" name="workingDirLabel">
      <property name="text">
       <string>Working directory</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="FileSelector" name="workingDirSelector"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="atomLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>96</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Sphere center</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="AtomSelector" name="atomSelector">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="4">
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="settings_button">
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_dock">
        <property name="text">
         <string>Dock!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0" colspan="4">
     <widget class="QPlainTextEdit" name="text">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileSelector</class>
   <extends>QLineEdit</extends>
   <header>widgets.file_selector.h</header>
  </customwidget>
  <customwidget>
   <class>AtomSelector</class>
   <extends>QLineEdit</extends>
   <header>widgets.atom_selector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
